mcr_perception_tools/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ find_package(catkin REQUIRED
 
 catkin_package(
   CATKIN_DEPENDS
-    mcr_object_recognition_bounding_box
+    # mcr_object_recognition_bounding_box
```

Comment:
If you want to remove this, why don't you delete the line instead of commenting it out (as in the `package.xml`)?

Reply:
yes, that's correct. I updated this in the latest commit.